### PR TITLE
Adding calculate function #29

### PR DESCRIPTION
**Pull request by hsguidi**
Split __main__.py in calculate.py and __main__.py

### CODE REVIEW

File: mathinterpreter/__main__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
-from mathinterpreter.interpreter import Interpreter
-from mathinterpreter.lexer import Lexer
-from mathinterpreter.parser import Parser
+#from mathinterpreter.interpreter import Interpreter
+#from mathinterpreter.lexer import Lexer
+#from mathinterpreter.parser import Parser
+from mathinterpreter.calculate import calculate
 
 
 def main():
```
Comment (aureliobarbosa) [inline mathinterpreter/__main__.py:7]: In my opinion there is no need anymore for a try block inside this main function, because the main problematic situations were move to function calculate(). Do you agree? If so, can you please remove this try-except block?

**Comment by aureliobarbosa** [comment]
Another issue. You need to test this function, think on a few simple cases. Do not forget to test an empty string, as an special edge case. If you are bored about it now, let's close the PR e open an issue. Then you fix it latter.
